crypto/rand/drbg_lib.c

```diff
@@ -380,16 +426,51 @@ int RAND_DRBG_instantiate(RAND_DRBG *drbg,
         max_entropylen += drbg->max_noncelen;
     }
 
+    /*
+     * If we're running with continuous RNG testing and haven't initialised
+     * yet, we need more data.  We add one block and round up to the nearest
+     * block boundary.  The entropy requirements are not altered.
```

Comment:
This comment makes me wonder whether it really makes sense to increase only the buffer size without increasing the entropy requirements. And thinking even further: does it make sense to use the "uncompressed" entropy input for the CRNG test? Shouldn't it rather be the "compressed" entropy input after applying the derivation function? (In the case where a df is used, which is our default.)

Reply:
I'll have to ask about this one.

Reply:
I didn't note the response from the FIPS guy: the raw input needs to be used, the entropy requirement doesn't increase.